### PR TITLE
Added JavaScript to the list and the required files to start adding content to the project

### PR DESCRIPTION
**Pull request by TheGameFreak720**

### CODE REVIEW

File: technologies/technologies.json [hunk 1]
```diff
@@ -11,6 +11,12 @@
         "displayName": "Laravel",
         "description": "Laravel is a free, open-source PHP web framework, created by Taylor Otwell and intended for the development of web applications following the model–view–controller (MVC) architectural pattern and based on Symfony."
     },
+    {
+        "id": "javascript",
+        "isVisible": true,
```
Comment (remojansen) [inline technologies/technologies.json]: If you don't mind I think it is better to keep `isVisible` as `false` until we have the list almost ready.

```suggestion
        "isVisible": false,
```

File: technologies/javascript/javascript.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+                ]
+            }
+        ],
+        "advanced_beginner": [],
+        "competent": [],
+        "proficient": [],
+        "expert": []
+    },
+    "notes": [
+        "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua."
```
Comment (remojansen) [inline technologies/javascript/javascript.json]: Please remove the placeholder note if you don't think that notes are necessary
```suggestion
```

**Comment by TheGameFreak720** [comment]
Sorry for that. Made changes that you suggested so we can merge the branch.

File: technologies/javascript/javascript.json (hunk 2 above)
Comment (TheGameFreak720) [reply technologies/javascript/javascript.json]: Change is done now

File: technologies/technologies.json (hunk 1 above)
Comment (TheGameFreak720) [reply technologies/technologies.json]: I set the isVisible flag to false

**Comment by remojansen** [comment]
:+1: Thanks for the PR

**Pull request merged**